samples/bot-conversation/nodejs/README.md

```diff
@@ -173,6 +173,18 @@ You can interact with this bot in Teams by sending it a message, or selecting a
 - A new screen will be open and the text will be read by default which is mentioned inside the speak property of adaptive card.
 ![immersive-reader-screen](Images/immersiveReaderScreen.png)
 
+ 5. **Message Update Events**
+- You will recieve event updates for message edit/delete features.
+
+- Message is edited
+![message-edit](Images/messageEditEvent.png)
+
+- Message is soft-deleted
+![message-soft-delete](Images/softDeleteMessage.png)
+
+- Message is restored
+![message-undelete](Images/messageUndelete.png)
+
 You can select an option from the command list by typing ```@TeamsConversationBot``` into the compose message area and ```What can I do?``` text above the compose area.
 
 ## Deploy the bot to Azure
```

Comment:
Please add SDK methods references or doc

Reply:
The official doc is not released yet publicly. We will add it when doc is published